### PR TITLE
Add parent aggregation

### PR DESCRIPTION
**Pull request by SosthenG**
I also updated the docker elasticsearch version to 6.6.1 to be able to launch the test.

### CODE REVIEW

File: env/elasticsearch/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #https://www.docker.elastic.co/
-FROM docker.elastic.co/elasticsearch/elasticsearch:6.5.2
+FROM docker.elastic.co/elasticsearch/elasticsearch:6.6.1
```
Comment (ruflin) [inline env/elasticsearch/Dockerfile]: Could you open a separate PR with this change to make sure it does not have any other affects? With this change, we normally also update the README and all the places where we mention what we test with (just search for 6.5.2)

Comment (SosthenG) [reply env/elasticsearch/Dockerfile]: Done in #1620

Comment (ruflin) [reply env/elasticsearch/Dockerfile]: Thanks, as soon as the other one is in, could you rebase on top of it?

Comment (SosthenG) [reply env/elasticsearch/Dockerfile]: Sure, I will do it!

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,11 +6,15 @@ All notable changes to this project will be documented in this file based on the
...
 ### Bugfixes
+* Always set the Guzzle `base_uri` to support connecting to multiple ES hosts. [#1618](https://github.com/ruflin/Elastica/pull/1618)
 
 ### Added
 
+* Added `ParentAggregation` [#1616](https://github.com/ruflin/Elastica/pull/1616)
+
 ### Improvements
 * Added `native_function_invocation` CS rule [#1606](https://github.com/ruflin/Elastica/pull/1606)
+* Elasticsearch test version changed from 6.5.2 to 6.6.1 [#1620](https://github.com/ruflin/Elastica/pull/1620)
```
Comment (ruflin) [inline CHANGELOG.md]: Seems like a bit more changes in the changelog then there should be?

File: README.md [hunk 3]
```diff
@@ -17,7 +17,7 @@ All library issues should go to the [issue tracker from github](https://github.c
 
 Compatibility
 -------------
-This release is compatible with all Elasticsearch 6.x releases. It was tested with version 6.5.2.
+This release is compatible with all Elasticsearch 6.x releases. It was tested with version 6.6.1.
```
Comment (ruflin) [inline README.md]: That should go away with a rebase? Also below?

File: CHANGELOG.md (hunk 2 above)
Comment (SosthenG) [reply CHANGELOG.md]: Sorry this is my first PR, I'm still discovering how to use git in such way...
I did a rebase of my parent-agg branch onto your origin/master branch but I don't really understand what happened after (my IDE added an empty merge, I don't know why, but I definitely did a rebase). When I try to rebase again, it says that my branch is up-to-date with your master, and for me it seems that way, I suppose I don't understand well what this rebase should have done.
Can you explain me what's wrong and how to fix it please? Really sorry for the inconvenience.

Comment (ruflin) [reply CHANGELOG.md]: One thing that potentially might have happened, is that your master is not up-to-date. Can you go to your master and pull the most recent version? You can use `git log` to see which commit your master is on. Today it should be https://github.com/ruflin/Elastica/commit/d0433d6e6deec8b1547cb4042d97042dcf00ebad (the id or the prefix of it you should see in your git log output on the top).

Comment (SosthenG) [reply CHANGELOG.md]: Actually, the first time I tried to rebase directly on your master, not mine (which was not sync with yours).
I tried to redo the operation with my master, after setting it in sync with yours.
It looks good to me this time, I hope it is :-)

**Comment by ruflin** [comment]
PR looks good, but there is a failing test. Unfortunately can't really see how it's related to this PR especially as only 1 PHP version failed: https://travis-ci.org/ruflin/Elastica/jobs/512448687

**Comment by ruflin** [comment]
I restarted the test to see if it's just flaky.

**Comment by ruflin** [comment]
@SosthenG Thanks for the contribution.

**Pull request merged**